main.tf

```diff
@@ -105,49 +103,106 @@ module "client" {
   source = "./modules/server"
 
-  enable_cai_bucket                                   = var.enable_cai_bucket
-  project_id                                          = var.project_id
-  gsuite_admin_email                                  = var.gsuite_admin_email
-  forseti_version                                     = var.forseti_version
-  forseti_repo_url                                    = var.forseti_repo_url
+  project_id               = var.project_id
+  forseti_version          = var.forseti_version
+  forseti_repo_url         = var.forseti_repo_url
+  forseti_home             = var.forseti_home
+  forseti_run_frequency    = var.forseti_run_frequency
+  server_type              = var.server_type
+  server_region            = var.server_region
+  server_boot_image        = var.server_boot_image
+  server_boot_disk_size    = var.server_boot_disk_size
+  server_boot_disk_type    = var.server_boot_disk_type
+  server_tags              = var.server_tags
+  server_access_config     = var.server_access_config
+  server_private           = var.server_private
+  cloudsql_proxy_arch      = var.cloudsql_proxy_arch
+  network                  = var.network
+  subnetwork               = var.subnetwork
+  network_project          = var.network_project
+  server_grpc_allow_ranges = var.server_grpc_allow_ranges
+  server_ssh_allow_ranges  = var.server_ssh_allow_ranges
+  server_instance_metadata = var.server_instance_metadata
+  suffix                   = local.random_hash
+
+  policy_library_home                    = var.policy_library_home
+  policy_library_repository_url          = var.policy_library_repository_url
+  policy_library_sync_enabled            = var.policy_library_sync_enabled
+  policy_library_sync_gcs_directory_name = var.policy_library_sync_gcs_directory_name
+  policy_library_sync_git_sync_tag       = var.policy_library_sync_git_sync_tag
+  policy_library_sync_ssh_known_hosts    = var.policy_library_sync_ssh_known_hosts
+
+  server_iam_module    = module.server_iam
+  cloudsql_module      = module.cloudsql
+  server_gcs_module    = module.server_gcs
+  server_rules_module  = module.server_rules
+  server_config_module = module.server_config
+  client_iam_module    = module.client_iam
+
+  services = google_project_service.main.*.service
+}
+
+module "cloudsql" {
+  source             = "./modules/cloudsql"
+  cloudsql_disk_size = var.cloudsql_disk_size
+  cloudsql_private   = var.cloudsql_private
+  cloudsql_region    = var.cloudsql_region
+  cloudsql_type      = var.cloudsql_type
+  network_project    = var.network_project
+  project_id         = var.project_id
+  services           = google_project_service.main.*.service
+  suffix             = local.random_hash
+}
+
+module "server_iam" {
+  source                  = "./modules/server_iam"
+  cscc_violations_enabled = var.cscc_violations_enabled
+  enable_write            = var.enable_write
+  project_id              = var.project_id
+  org_id                  = var.org_id
```

Comment:
Do you need a parameter for `folder_id` here as well?

Reply:
yikes.  yes.